common/services/catalogue/filters.ts

```diff
@@ -131,7 +131,7 @@ const subjectsFilter = ({
 const genresFilter = ({ works, props }: WorksFilterProps): CheckboxFilter => ({
   type: 'checkbox',
   id: 'genres.label',
-  label: 'Genres',
+  label: 'Type/Technique',
```

Comment:
Should this not be plural to match the other filters? Ie, `Types/Techniques`.

Reply:
I was just about to ask - this is what we have on the works page
<img width="771" alt="Screenshot 2021-04-14 at 13 20 32" src="https://user-images.githubusercontent.com/31692/114709238-3c3dd200-9d24-11eb-94d8-e117b20bae6e.png">

But just realised this is additive. Will update.

Reply:
Assuming we use the singular on the work page as we _often_ have 1. But can have multiple, so probably worth pluralising the heading there too potentially (although I doubt it's a problem for anyone, just trying to make sure the filter / aggs sync work carries across).